x-pack/plugins/cases/server/common/types/connectors.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export type ConnectorFieldsPersistedAttributes = Array<{
```

Comment:
nit: What do you think about `CasePersistedConnectorFields`?

Reply:
How do you feel about dropping the `Case` part? I think it's kind of redundant.

Reply:
What about `ConnectorPersistedFields`? I am fine with all options 🙂.